### PR TITLE
Externalize Operations controller resources

### PR DESCRIPTION
**Pull request by NickyMateev**
**Description**

Changes proposed in this pull request:
- ...

**Related issue(s)**

- ...

**Pull Request status**

- [ ] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -89,7 +89,7 @@ global:
       version: "PR-2077"
     operations_controller:
       dir:
-      version: "PR-2063"
+      version: "PR-2106"
```
Comment (desislavaa) [inline chart/compass/values.yaml:92]: no need for this, there are no code changes

**Review by desislavaa — APPROVED** [review]

**Review by KirilKabakchiev — APPROVED** [review]

**Pull request merged**